lib/registrant.rb

```diff
@@ -0,0 +1,26 @@
+class Registrant 
+    attr_reader :name, :age, :permit, :license_data
+
+    def initialize(name, age, permit, license_data)
```

Comment:
We don't need license_data here in the arguments since it's just an attribute we're setting for every Registrant object. We also wanted permit to have a default value so line 4 would like like this: `def initialize(name, age, permit = false)`